test/parallel/test-coverage-with-inspector-disabled.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+require('../common');
+const fixtures = require('../common/fixtures');
+
+const assert = require('assert');
+const { spawnSync } = require('child_process');
+const env = Object.assign({}, process.env, { NODE_V8_COVERAGE: '/foo/bar' });
+const childPath = fixtures.path('v8-coverage/subprocess');
+const { status, stderr } = spawnSync(
+  process.execPath,
+  [childPath],
+  { env: env }
+);
+
+const warningMessage = 'The inspector is disabled, ' +
+                        'coverage could not be collected';
+
+assert.strictEqual(status, 0);
+assert.strictEqual(
+  stderr.toString().includes(`Warning: ${warningMessage}`),
+  true
```

Comment:
If you want a more helpful error message in case of failures (like you are seeing in Travis CI), you can add a third argument to `assert.strictEqual()`, e.g. `` `stderr: <${stderr}>` ``, that prints out the full contents of `stderr`.